packages/Alert/src/alert.scss

```diff
@@ -14,34 +17,86 @@
   --ids-alert-box-shadow: #{tokens.$shadow-6};
```

Comment:
Hopper has 4 shadows. Where is that coming from?

Reply:
this is not hopper